app/api/v1/endpoints/insurance.py

```
@@ -0,0 +1,66 @@
+async def read_insurances_by_disease(
+    disease_id: UUID,
+    db: AsyncSession = Depends(get_db)
+):
+    insurances = await insurance_crud.get_insurances_by_disease(db, disease_id)
+    return [InsuranceSchema.model_validate(insurance) for insurance in insurances]
+
+@router.post("/disease/{disease_id}", response_model=InsuranceSchema)
+async def create_insurance(
+    disease_id: UUID,
+    insurance: InsuranceCreate,
+    db: AsyncSession = Depends(get_db)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)